dipy/denoise/localpca.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import scipy as sp
+
+def localpca(arr, sigma, patch_radius=1, tou=0, rician=True):
+    '''
+    Local PCA Based Denoising of Diffusion Datasets
```

Comment:
Add the reference in first line description of function: "Local PCA Based Denoising of Diffusion MRI Datasets [1]_"